packages/shopify-codemod/test/fixtures/arguments-to-args-spread/sanity.input.js

```diff
@@ -0,0 +1,26 @@
+function argNamesShouldPreventSpread(arg1, arg2) {
```

Comment:
Could use some non-arrow function IIFE examples to show how they behave differently.